Cornelica/scenario/014_03a.txt

```diff
@@ -18,24 +18,24 @@
 @huki b=2 id=3
 @bs f=ユスティーナ@笑み body=-1 face=0 x=350 y=50 t=200
 
-[ユスティーナ]「あら、リリィちゃんじゃない。ふふ、男の子つれちゃって、デート中かしら？」 
+[Justina]"Oh, if it isn't Lily, hehe, you're with a boy, \nare you on a date?"
 
 @huki b=1 id=AC0
 @bs f=ユスティーナ@微笑 body=-1 face=0 x=350 y=50 t=100 wt=0
 @bs f=リリィ@焦り_r body=-1 face=0 x=-350 y=50 t=200
 
-[リリィ]「ア、アルはそんなんじゃないよ！」 
+[Lily]"Al isn't like that!"
```

Comment:
Referenced this way back for 013_03a I believe.

Reply:
Yep, similar